CMakeLists.txt

```diff
@@ -78,9 +78,9 @@ list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/Windows")
 #------------------------------------------------------------------------------
 # SmartPeak Version Information
 #------------------------------------------------------------------------------
-set (SMARTPEAK_PACKAGE_VERSION_MAJOR 1)
-set (SMARTPEAK_PACKAGE_VERSION_MINOR 0)
-set (SMARTPEAK_PACKAGE_VERSION_PATCH 0)
+set(SMARTPEAK_PACKAGE_VERSION_MAJOR "0" CACHE STRING "SmartPeak major version")
+set(SMARTPEAK_PACKAGE_VERSION_MINOR "0" CACHE STRING "SmartPeak minor version")
+set(SMARTPEAK_PACKAGE_VERSION_PATCH "0" CACHE STRING "SmartPeak patch version")
```

Comment:
how are the version numbers carried out to the respective variables?

Reply:
I think we will pass these variables while lunching cmake config. By default version is `0.0.0` which means no version defined

Reply:
Please elaborate what you would like to modify

Reply:
> I think we will pass these variables while lunching cmake config. By default version is `0.0.0` which means no version defined

could you provide a link to the cmake config file you are referring to?:)